package.json

```diff
@@ -61,6 +61,7 @@
     "json5": "2.2.3",
     "lodash": "4.17.21",
     "marked": "15.0.11",
+    "mongodb-schema": "^12.6.2",
```

Comment:
- The package must be moved to `devDependencies` if it's used  only as a CLI.
- The version number must be pinned.

From the PR description: run the native command provided by the packaeg without using `npx`, which may attempt to install the package again and potential cause trouble with the packages installed by our package manager, `pnpm`.